OPHD/Map/TileMap.h

```diff
@@ -137,7 +137,7 @@ class TileMap: public micropather::Graph
 	NAS2D::Timer mTimer;
 
 	NAS2D::Point<int> mMousePosition; /**< Current mouse position. */
-	NAS2D::Point<int> mMapHighlight; /**< Tile the mouse is pointing to. */
+	NAS2D::Vector<int> mMapHighlight; /**< Tile the mouse is pointing to. */
```

Comment:
Curious why the change here was made? It would seem a Point is a more natural representation of a specific tile?